src/main/java/de/mediathekview/mserver/crawler/srf/tasks/SrfSendungOverviewPageTask.java

```diff
@@ -2,41 +2,89 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mserver.base.utils.UrlUtils;
 import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
 import de.mediathekview.mserver.crawler.basic.AbstractRestTask;
 import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
 import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
 import de.mediathekview.mserver.crawler.srf.parser.SrfSendungOverviewDTO;
 import de.mediathekview.mserver.crawler.srf.parser.SrfSendungOverviewJsonDeserializer;
+import java.lang.reflect.Type;
 import java.util.Optional;
+import java.util.Set;
 import java.util.concurrent.ConcurrentLinkedQueue;
 import javax.ws.rs.client.Invocation;
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
-public class SrfSendungOverviewPageTask extends AbstractRestTask<SrfSendungOverviewDTO, CrawlerUrlDTO> {
+public class SrfSendungOverviewPageTask extends AbstractRestTask<CrawlerUrlDTO, CrawlerUrlDTO> {
 
+  private static final Logger LOG = LogManager.getLogger(SrfSendungOverviewPageTask.class);
+  
   private static final String ENCODING_GZIP = "gzip";
   private static final String HEADER_ACCEPT_ENCODING = "Accept-Encoding";
   
+  private static final Type OPTIONAL_DTO_TYPE_TOKEN = new TypeToken<Optional<SrfSendungOverviewDTO>>() {}.getType();
+
+  private final int pageNumber;
+  
+
   public SrfSendungOverviewPageTask(AbstractCrawler aCrawler, ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDTOs) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDTOs" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)